### DISCUSSION TITLE
关于pyarmor_runtime.so加载的问题

### DISCUSSION THREAD

**Discussion by yunkai-jia** [dashingsoft/pyarmor#1186] [Q&A] [answered]
我在想有没有一种可能是pyarmor_runtime.so在mac中自带的python解释器中可以去加载，但是在conda中的python解释器无法加载这个pyarmor_runtime.so？ 谢谢老师

**Comment by jondy** [comment, marked as answer]
pyarmor_runtime.so 就是一个普通的扩展模块，你如果不明白什么是扩展模块，请参阅 Python 的文档。

解决这个问题的核心在与 conda 支持不支持CPython 编写的扩展模块

问题在于我对conda也不了解，也不会去学习和了解这个工具，我也不可能去安装和学习每一个客户使用的工具包，所以这个问题我无法回答你。

你可以查阅 conda 的文档，看看 conda 支持不支持 扩展模块。如果支持，那么通过配置就可以装载，如果 conda 不支持 CPython 的扩展模块，那么 conda 就无法装载。

**Comment by yunkai-jia** [comment]
好的，谢谢老师啦，我去研究下
